other-markdown.Rmd

```diff
@@ -34,93 +42,206 @@ I normally structure my `README` as follows:
+
+# foofy 1.0.0
+
+## Major changes
+
+* Can now work with all grooveable grobbles!
+
+## Minor improvements and bug fixes
+
+* Printing scrobbles no longer errors (@githubusername, #100).
+
+* Wibbles are now 55% less jibbly (#200).
+```
+
+The example above demonstrates some organizing principles for `NEWS.md`:
+
+-   Use a top-level heading for each version: e.g. `# somepackage 1.0.0`.
     The most recent version should go at the top.
-    (pkgdown supports a few other formats, see more at <https://pkgdown.r-lib.org/reference/build_news.html>)
+    Typically the top-most entry in `NEWS.md` of your source package will read `# somepackage (development version)`.[^other-markdown-2]
 
--   Each change should be included in a bulleted list.
-    If you have a lot of changes you might want to break them up using subheadings, `## Major changes`, `## Bug fixes` etc.
-    I usually stick with a simple list until just before releasing the package when I'll reorganise into sections, if needed.
+-   Each change should be part of a bulleted list.
+    If you have a lot of changes, you might want to break them up using subheadings, `## Major changes`, `## Bug fixes`, etc.
+    We usually stick with a simple list until we're close to a release, at which point we organize into sections and refine the text.
     It's hard to know in advance exactly what sections you'll need.
+    The release checklist placed by `usethis::use_release_issue()` includes a reminder to polish the `NEWS.md` file.
 
 -   If an item is related to an issue in GitHub, include the issue number in parentheses, e.g. `(#​10)`.
     If an item is related to a pull request, include the pull request number and the author, e.g. `(#​101, @hadley)`.
-    Doing this makes it easy to navigate to the relevant issues on GitHub.
-
-The main challenge with `NEWS.md` is getting into the habit of noting a change as you make a change.
-
-## Other GitHub files
+    This helps an interested reader to find relevant context on GitHub and, in your pkgdown site, these issue and pull request numbers and usernames will be hyperlinks.
+    We generally omit the username if the contributor is already recorded in `DESCRIPTION`.
 
-You'll also find a number of other files in the `.github` directory.
-These are mostly useful for GitHub but are also supported by pkgdown:
+[^other-markdown-2]: pkgdown supports a few other wording choices for these headings, see more at <https://pkgdown.r-lib.org/reference/build_news.html>.
 
--   `LICENSE.md`: complete text of the license of your package. Discussed in ...
--   `cran-comments.md`: sent to CRAN when you submit a package. Discussed in ...
--   `SUPPORT.md`:
--   `CONTRIBUTING.md`:
--   `CODE_OF_CONDUCT.md`:
--   `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE`
--   
+The main challenge with `NEWS.md` is getting into the habit of noting any user-visible change when you make it.
+It's especially easy to forget this when accepting external contributions.
```

Comment:
Might be worth a brief comment that commit messages should be developed facing and NEWS bullets user facing?

Reply:
added